engine/src/main/java/org/hibernate/search/query/engine/impl/AbstractHSQuery.java

```diff
@@ -530,6 +535,25 @@ private FacetMetadata findFacetMetadata(FacetingRequest facetingRequest, Iterabl
 		return facetMetadata;
 	}
 
+	protected List<IndexManager> getIndexManagers(EntityIndexBinding binding) {
+		FullTextFilterImplementor[] fullTextFilters = getFullTextFilterImplementorsAsArray();
+		List<IndexManager> indexManagers = Arrays.asList( binding.getSelectionStrategy().getIndexManagersForQuery( fullTextFilters ) );
+		return indexManagers;
+	}
+
+	private FullTextFilterImplementor[] getFullTextFilterImplementorsAsArray() {
```

Comment:
Is the fact that it returns an Array really important? I wouldn't include it in the method name.

And I must admit I preferred the old method name altogether (`getFullTextFilter()`). It's more to the point and doesn't add useless information.